hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/utilops/records/SnapshotModeOp.java

```diff
@@ -159,13 +159,23 @@ public class SnapshotModeOp extends UtilOp implements SnapshotOp {
 
     public static void main(String... args) throws IOException {
         // Helper to review the snapshot saved for a particular HapiSuite-HapiSpec combination
-        final var snapshotFileMeta = new SnapshotFileMeta("CryptoTransfer", "okToRepeatSerialNumbersInBurnList");
-        final var snapshot = loadSnapshotFor(PROJECT_ROOT_SNAPSHOT_RESOURCES_LOC, snapshotFileMeta);
+        final var snapshotFileMeta = new SnapshotFileMeta("ContractCall", "MultipleSelfDestructsAreSafe");
+        final var maybeSnapshot = suiteSnapshotsFrom(
+                        resourceLocOf(PROJECT_ROOT_SNAPSHOT_RESOURCES_LOC, snapshotFileMeta.suiteName()))
+                .flatMap(
+                        suiteSnapshots -> Optional.ofNullable(suiteSnapshots.getSnapshot(snapshotFileMeta.specName())));
+        if (maybeSnapshot.isEmpty()) {
+            throw new IllegalStateException("No such snapshot");
+        }
+        final var snapshot = maybeSnapshot.get();
         final var items = snapshot.parsedItems();
+        final var dumpLoc = Files.newBufferedWriter(Paths.get(snapshotFileMeta + ".txt"));
```

Comment:
Also given this usage in a utility CLI I honestly don't care but IntelliJ points out that this file should be opened under a try-with-resources.

Reply:
Ah interestingly this _is_ important, as a `BufferedWriter` will not necessarily flush its contents to the file without it! 

Fixed.

Reply:
oops!  now I Iearned something too, thanks!